Springer Link.js

```diff
@@ -197,19 +197,15 @@ function complementItem(doc, item) {
 
 		item.abstractNote = abstract.trim();
 	} else {
-		let absSections = ZU.xpath(doc, '//*[(@id = "Abs2-content")]//p');
-		let sectionTitles = ZU.xpath(doc, '//*[(@id = "Abs2-content")]//*[contains(concat( " ", @class, " " ), concat( " ", "c-article__sub-heading", " " ))]');
-		let titleTextGerman = ZU.xpathText(doc, '//*[(@id = "Abs1-content")]//p');
+		let otherAbstracts = doc.querySelectorAll('#Abs2-section');
```

Comment:
How can we ensure that there are no regression problems, i.e. that the now removed code is not needed in another context?

Reply:
i run the test for all 15 urls and with the previous version i got wrong abstract e.g. https://link.springer.com/article/10.1007/s00481-021-00606-5

Reply:
One then might change the order of evaluation, but ok then we go with it.